sashimi/camera.py

```diff
@@ -1,7 +1,7 @@
 from multiprocessing import Process, Queue
 from enum import Enum
 from arrayqueues.shared_arrays import ArrayQueue
-from sashimi.hardware.cameras.Hamamatsu_ORCA_flash_wrapper import HamamatsuOrcaFlashCamera
+from sashimi.hardware.cameras.Hamamatsu_wrapper import HamamatsuCamera
```

Comment:
lower case please, even Hamamatsu does not have the honor of breaking Python noventions :)